packages/gatsby-remark-images/src/index.js

```diff
@@ -185,9 +213,7 @@ module.exports = (
   >
     <span
       class="${imageBackgroundClass}"
-      style="padding-bottom: ${ratio}; position: relative; bottom: 0; left: 0; background-image: url('${
-      fluidResult.base64
-    }'); background-size: cover; display: block;"
+      style="padding-bottom: ${ratio}; position: relative; bottom: 0; left: 0; background-image: url(&quot;${placeholderImageData}&quot;); background-size: cover; display: block;"
```

Comment:
`&quot;` I don't think you can use a html entity here.

Reply:
This is the only way I have found to use the [optimized SVG](https://codepen.io/tigt/post/optimizing-svgs-in-data-uris) that `traceSVG` returns in the style attribute. Based on [this stackoverflow post](https://stackoverflow.com/questions/5352947/how-to-escape-quotes-in-inline-styles/5352973#5352973) and [the HTML spec](https://www.w3.org/TR/html4/intro/sgmltut.html#h-3.2.2) I believe the HTML entity is valid anywhere in an HTML document. It has worked as far as I have seen in my testing.

Using double quotes around the SVG data conflicts with the style attribute's quotes. Using single quotes around the SVG data conflicts with the single quotes in the optimized SVG encoding. Using single quotes for the style attribute as a whole with double quotes around the data also conflicts with the single quotes in the optimized SVG encoding. Using no quotes around the SVG data does not work (I believe because of spaces in the data). I have not thought of any other possible solution but I am open to suggestions.

Reply:
Thanks for the deep explanation.

What about escaping singles quotes in the SVG image?

```
const escapeSingleQuotes = (value) => value.replace(/'/g, '\\\'');
```
```
<div style="
    background-image: url('${escapeSingleQuotes(placeholderImageData)}');
    background-size: cover;
"
></div>
```

Reply:
Thanks for the suggestion. It definitely is a cleaner solution. I just pushed a commit along these lines. Apparently webpack or something else is optimizing it along the way, so to make things even better there was no increase in the number of bytes sent to the browser even with the escaped quotes.